tests/python/shared/fixtures/init.py

```diff
@@ -205,7 +223,7 @@ def kube_restore_data_volumes():
     pod_name = _kube_get_server_pod_name()
     kube_cp(
         CVAT_DB_DIR / "cvat_data.tar.bz2",
-        f"{pod_name}:/tmp/cvat_data.tar.bz2",
+        f"{pod_name}:/tifmp/cvat_data.tar.bz2",
```

Comment:
@zhiltsov-max , is it typo?

Reply:
Yes, fixed. Sometimes I occasionally can paste something with my laptop's touchpad.